.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
```

Comment:
```suggestion
default_language_version:
  python: python3

repos:
```

I've found that this helps prevent issues where pre-commit hooks were installed with a particular Python version and the user later changed the Python version. Just a suggestion.

Reply:
Sounds good. I don't think this applies to any of the current hooks but if we add some python specific ones in the future this will make sure that we're enforcing python3 syntax.